### PR TITLE
[Bugfix] 2019 Season Fixes

### PR DESCRIPTION
**Pull request by crsharff**
Added FC Cincinnati to the list of teams which requires an increment of the number of teams in the eastern conference. Fixed an issue with retrieving "Recent Match Results" when we are less than 9 games into the season.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def matchesTable(table, str):
 	,{'link': '/r/Rapids', 'contains': 'Colorado Rapids'}
 	,{'link': '/r/minnesotaunited', 'contains': 'Minnesota United'}
 	,{'link': '/r/LAFC', 'contains': 'Los Angeles Football Club'}
+	,{'link': '/r/FCCincinnati', 'contains': 'FC Cincinnati'}
```
Comment (Newman29) [inline main.py]: Isn't Cincinnati in the Eastern conference and this is for the western conference standings? (Just reading over the PR, haven't looked too much into the code)

Comment (crsharff) [reply main.py]: Nice catch, yeah you might be right, this was one of my early changes and it likely isn't necessary. Let me test without just to make sure.

File: main.py [hunk 2]
```diff
@@ -320,7 +321,9 @@ def parseSchedule():
 		strListGames.append("Date|||Opponent|Result\n")
 		strListGames.append(":---:|:---:|---|:---|:---:|:---:\n")
 
-		for game in teamGames[nextGameIndex-9:nextGameIndex]:
```
Comment (kinigitbyday) [inline main.py]: This isn't actually a bug, is it? 
In Python 2.7,
```
>>> e = [1,2,3,4]
>>> e[-7:2]
[1, 2]
```

Comment (crsharff) [reply main.py]: Interesting, did the behavior of slice change in Python 3.x? I'll test this when I get back from lunch.

Comment (Newman29) [reply main.py]: I was able to verify the same behavior in 3.7

Comment (Newman29) [reply main.py]: Taking a closer look at it I do think this is the fix. Haven't touched Python in a while, and forgot that you can start counting from a negative index 

image

Comment (kinigitbyday) [reply main.py]: @Newman29 Oh, that's right. It does weird wrap around stuff. And I just happened to use a set of somewhat random values that looked right.

Comment (kinigitbyday) [reply main.py]: @crsharff sorry about the false alarm. Your changes look good, and should fix those issues.

Comment (Newman29) [reply main.py]: I too had forgotten about it. I just happened to be touching up on Python for work and stumbled across exact thing

Comment (Newman29) [reply main.py]: @nikdahl this should fix the problem if you want to approve the PR and merge it in and get it deployed (if no one else has resolved it).

In the worst case scenario, it won't make anything any worse ¯\_(ツ)_/¯

Comment (crsharff) [reply main.py]: Yeah I think the easiest solution (for now) is to cap the slicing at 0 and avoid it going negative.

Comment (crsharff) [reply main.py]: Also, no worries @kinigitbyday, thanks for looking over it - that's what PR's are for ;)

**Comment by crsharff** [comment]
@nikdahl Did you want to get this PR pulled in, looks to be ready to go.

**Pull request merged**